OSDOCS-9433: getting cluster id section is added to microshift support (#73364)
openshift/openshift-docs: modules/microshift-get-nonrunning-cluster-id-kubesystem.adoc

// Module included in the following assemblies:
//
// microshift_support/microshift-getting-cluster-id.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-get-nonrunning-cluster-id-kubesystem_{context}"]
= Getting the cluster ID of a nonrunning cluster

For a cluster that is not running, you can get the cluster ID from the `cluster-id` file found in `/var/lib/microshift` directory.

.Procedure

. Get the cluster ID of the cluster that is not running by entering the following command:
+
[source,terminal]
----
$ sudo cat /var/lib/microshift/cluster-id
----
.Example output
+
[source,terminal]
----
7cf13853-68f4-454e-8f5c-1af748cbfb1a
----
+
[NOTE]
====
The {microshift-short} service copies the `kube-system` namespace metadata UID to the `cluster-id` file so that the UID is accessible when the cluster is not running. You can find the `cluster-id` file in the `/var/lib/microshift` directory.
====

. You can view the {microshift-short} database files in the `/var/lib/microshift` directory by entering the following command:
+
[source,terminal]
----

> $ sudo ls -l /var/lib/microshift

Reviewer: ```suggestion
$ sudo ls -l /var/lib/microshift/cluster-id
```

// Module included in the following assemblies:
//
// microshift_support/microshift-getting-cluster-id.adoc

:_mod-docs-content-type: PROCEDURE
[id="microshift-get-nonrunning-cluster-id-kubesystem_{context}"]
= Getting the cluster ID of a nonrunning cluster

For a cluster that is not running, you can get the cluster ID from the `cluster-id` file found in `/var/lib/microshift` directory.

.Procedure

> . Get the namespace UID of a cluster that is not running by entering the following command:

Reviewer: ```suggestion
. Get the non-running cluster ID by entering the following command:
```

Author: done the change

+
[source,terminal]
----
$ sudo cat /var/lib/microshift/cluster-id
----
.Example output
+
[source,terminal]
----
7cf13853-68f4-454e-8f5c-1af748cbfb1a
----
+
[NOTE]
====
The {microshift-short} service copies the `kube-system` namespace metadata UID to the `cluster-id` file so that the UID is accessible when the cluster is not running. You can find the `cluster-id` file in the `/var/lib/microshift` directory.
====

. You can view the {microshift-short} database files in the `/var/lib/microshift` directory by entering the following command:
+
[source,terminal]
----
$ sudo ls -l /var/lib/microshift
----
.Example output
+
[source,terminal]
----
total 16
drwxr-xr-x. 14 root root 4096 Jan 28 12:03 certs
-r--------.  1 root root   36 Jan 28 12:04 cluster-id
drwx------.  3 root root   20 Jan 28 12:03 etcd
drwxr-xr-x.  3 root root   20 Jan 28 12:04 kubelet-plugins
drwxr-xr-x. 10 root root 4096 Jan 28 12:03 resources
-rw-------.  1 root root   65 Jan 28 12:03 version

> ----

Reviewer: I do not think this is necessary anymore. The note above gives enough information. 
In case we insist to have some directory listing, let's only specify the `cluster-id` file, but I prefer not to have it at all.

```
$ sudo ls -l /var/lib/microshift/cluster-id
-r-------- 1 root root 36 Mar  7 11:49 /var/lib/microshift/cluster-id
```

Author: Done the change
